CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [1.2.2] - 2022-04-07
+### Added
+- SPIRE Server and Agent log files can be rotated by sending the `SIGUSR2` signal to the process (#2703)
+- Entries created by k8s-workload-registrar in CRD mode set the `downstream` field when the corresponding field is set in the SpiffeID CRD object (#2885)
+- Small documentation improvements (#2851)
+### Fixed
+- The `k8sbundle` Notifier plugin in SPIRE Server no longer consumes excessive CPU cycles (#2857)
+- SPIRE can now be compiled on macOS machines with an Apple Silicon CPU (#2876)
+### Security
```

Comment:
Hmm. I think this probably goes unto "Changed". I think security should be more or less reserved for CVE related things.

Reply:
Yeah, I originally put it in "Changed" but then thought maybe "Security" was appropriate since it _technically_ has a security impact. I think it's probably appropriate to put it in either, but "Changed" is better since the security impact is pretty small here.